### ISSUE TITLE
Auto Code

### ISSUE THREAD

**Issue by GabriellaYang** [FalconX-Robotics/frc2018#8]
So this code is not compatible with the new encoders that we are using with the talons? We also need to set up the pigeon like really, really soon or we will be stuck using the old gyroscope for the commands which tends to drift quite a bit. COOOOL THOUGH, our robot is mucho spicy kimchi

**Comment by moreheadm** [comment]
Yes someone should read and understand the pigeon documentation.
